test-cases/tests/alerts/c14-verify-pvc-alerts.md

```diff
@@ -0,0 +1,14 @@
+---
+targets:
+  - 2.6.0
+---
+
+# C14 - Verify PVC alerts
+
+## Description
+
+More info: <https://issues.redhat.com/browse/INTLY-8413>
+
+## Steps
+
+TBD
```

Comment:
@grdryn can you provide some verifications steps for this test case?

Reply:
Sure!

1. Open a shell in a pod that has a PV attached
2. Use `fallocate` to make a file big enough to bring it past 97% full, but maybe less than 100% so that you don't cause other problems
3. See that the alerts fire
4. Delete your fallocated file to clean up

E.g. for the fuse-prometheus pod in the redhat-rhmi-fuse namespace:

Try to figure out how much space you'll need to get to 98%
```
sh-4.2$ df -h /prometheus/
Filesystem      Size  Used Avail Use% Mounted on
/dev/nvme3n1    9.8G  293M  9.5G   3% /prometheus
```

Might take a little trial and error to get into the right range:
```
sh-4.2$ fallocate -l 9250M /prometheus/big-test-file
sh-4.2$ df -h /prometheus/
Filesystem      Size  Used Avail Use% Mounted on
/dev/nvme3n1                          9.8G  9.6G  207M  98% /prometheus
```

...check prometheus to see the alert go to pending (might take a short period of time)...

Clean up file:
```
sh-4.2$ rm /prometheus/big-test-file
sh-4.2$ df -h /prometheus/
Filesystem      Size  Used Avail Use% Mounted on
/dev/nvme3n1    9.8G  293M  9.5G   3% /prometheus
```

Reply:
@obrienrobert @HubertStefanski FYI

Reply:
this seems pretty straight forward, I think we could actually manage to automate this test case pretty quickly, I think some simple maths could also bring this to a less `trial and error` method, i.e `((PVGi - ALGi) / 100) * 97` where PVGi is the total size of the PV and ALGi is the allocated (used) space, this should get us to the right range pretty much spot on

Reply:
@HubertStefanski I presume that this test is just being considered a once-off test for this release, rather than one of the regular release tests. If that's the case, then it's not worth trying to automate.

@jhellar am I correct that this will just be a once-off test for this release?

Reply:
In that case ignore what I said :laughing: , but the maths should check out for anyone that does run this though

Reply:
@grdryn thanks for the steps, yes, this would be just once-off test